localization/vision_common/include/vision_common/feature_match.h

```diff
@@ -0,0 +1,38 @@
+/* Copyright (c) 2017, United States Government, as represented by the
+ * Administrator of the National Aeronautics and Space Administration.
+ *
+ * All rights reserved.
+ *
+ * The Astrobee platform is licensed under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with the
+ * License. You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
```

Comment:
A lot of the code in this package looks similar to things in https://github.com/nasa/astrobee/tree/master/localization/interest_point/include/interest_point and https://github.com/nasa/astrobee/tree/master/localization/lk_optical_flow . Any thoughts on merging them / using this there?

Reply:
Yeah I think that makes sense.  The lk_optical_flow_feature_detector_and_matcher class should provide most if not all the options used in the current lk_optical_flow, and it might be cool to make lk_optical_flow into a more generic feature tracker using the new brisk and surf detector_and_matcher classes (which probably wouldn't be used by us but could be interesting offline options or useful to others using our code).  I can look into this for a future pr